plugins/package-managers/bazel/src/main/kotlin/Bazel.kt

```diff
@@ -58,6 +58,7 @@ import org.ossreviewtoolkit.utils.common.withoutPrefix
 import org.semver4j.RangesList
 import org.semver4j.RangesListFactory
 
+private const val DEFAULT_BAZEL_VERSION = "7.0.1"
```

Comment:
I believe it would be nice to name this just `BAZEL_FALLBACK_VERSION`.
This aligns with the env var name. The term "default" indicated that there is some configuration which 
allows deviating from the default, which is not the case-